decidim-core/lib/tasks/upgrade/decidim_migrate_wysiwyg_content.rake

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+namespace :decidim do
+  namespace :upgrade do
+    desc "Registers records for the WYSIWYG content migration"
+    task :register_wysiwyg_migration do
+      # Core
+      Decidim::Upgrade::WysiwygMigrator.register_model(
+        "Decidim::Organization",
+        [:description, :omnipresent_banner_short_description, :highlighted_content_banner_short_description, :welcome_notification_body, :admin_terms_of_use_body]
```

Comment:
```suggestion
        [:description, :omnipresent_banner_short_description, :highlighted_content_banner_short_description, :welcome_notification_body, :admin_terms_of_service_body]
```

Reply:
This is fixed with the latest changes. I also added specs for the migration rake tasks.